apiserver/observer/metricobserver/metricobserver.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package metricobserver
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/errors"
+	"github.com/juju/utils/clock"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/juju/juju/apiserver/observer"
+	"github.com/juju/juju/rpc"
+)
+
+const (
+	facadeLabelIndex = iota
```

Comment:
Yep, those are unnecessary. Some of the Prometheus APIs expect the label names/values in a certain order, but we're not using those APIs. Updated as suggested.